pingdom/team_types_test.go

```diff
@@ -0,0 +1,54 @@
+package pingdom
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTeamPutParams(t *testing.T) {
+	team := TeamData{
+		Name:    "fake team",
+		UserIDs: "1,2",
+	}
+	params := team.PutParams()
+	want := map[string]string{
+		"name":    "fake team",
+		"userids": "1,2",
+	}
+
+	assert.Equal(t, want, params, "Team.PutParams() should return correct map")
+}
+
+func TestTeamPostParams(t *testing.T) {
+	team := TeamData{
+		Name:    "fake team",
+		UserIDs: "",
+	}
+	params := team.PostParams()
+	want := map[string]string{
+		"name": "fake team",
+	}
+
+	assert.Equal(t, want, params, "Team.PostParams() should return correct map")
+}
+
+func TestTeamValid(t *testing.T) {
+	team := TeamData{
+		Name:    "fake team",
+		UserIDs: "",
+	}
+	params := team.Valid()
+
+	assert.Equal(t, nil, params, "Maintenance.Valid() should return nil if valid")
```

Comment:
Could you change the message here from `Maintenance` to `Team`?

Reply:
Done